test/frontend-integration-test/selenium-standalone-chrome-gcloud-nodejs.Docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM selenium/standalone-chrome
+
+USER root
+
+ARG CLOUD_SDK_VERSION=244.0.0
+ENV CLOUD_SDK_VERSION=$CLOUD_SDK_VERSION
+
+ARG INSTALL_COMPONENTS=kubectl
+ENV PATH "$PATH:/opt/google-cloud-sdk/bin/"
+RUN apt-get update -qqy && apt-get install -qqy \
+        curl \
+        gcc \
+        python-dev \
+        python-setuptools \
+        apt-transport-https \
+        lsb-release \
+        openssh-client \
+        git \
+        gnupg \
+    && easy_install -U pip && \
+    pip install -U crcmod && \
+    export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && \
+    echo "deb https://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" > /etc/apt/sources.list.d/google-cloud-sdk.list && \
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
+    apt-get update && apt-get install -y google-cloud-sdk=${CLOUD_SDK_VERSION}-0 $INSTALL_COMPONENTS
+
+
+RUN apt-get update -y && \
+    curl --silent --show-error --location https://deb.nodesource.com/setup_8.x | bash - && \
+    apt-get install -y nodejs && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /usr/share/locale/* /usr/share/i18n/locales/*
```

Comment:
This isn't going to change the image size at this point. It needs to be merged with the previous layer.

Reply:
That's true >_<